ui-spacetimechart/src/components/workSchedules/Region.tsx

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useDraw } from '../../hooks/useCanvas';
+import { type DrawingFunction } from '../../lib/types';
+import { type SpaceTimeRegion } from '../../types';
+
+const Region = ({
+  timeStart, // x date timestamp
+  timeEnd, // x date timestamp
+  spaceStart, // y metre
+  spaceEnd, // y metre
+  image,
+}: SpaceTimeRegion) => {
+  const [imageElement, setImageElement] = useState<HTMLImageElement | null>(null);
+  useEffect(() => {
+    if (image) {
+      const newImage = new Image();
```

Comment:
Nit: `newImage` doesn't really convey the difference between this variable and `image`. `image` is the URL and `newImage` is the `Image` object. Maybe rename `image` to `imageUrl` instead?

Reply:
done